### DISCUSSION TITLE
Project page workflow

### DISCUSSION THREAD

**Discussion by Reijnn** [community/community#21940] [New to GitHub]
<p>Hey everyone, our team has decided to move completely to GitHub for our project management. Including using the repository project page. Our workflow is like this:</p>
<ol>
<li>
<p>Create a project card with a user story.</p>
</li>
<li>
<p>Create an issue from this card and use a template to fill out the information.</p>
</li>
<li>
<p>Create a feature branch, submit some initial work and create a pull request asap.</p>
</li>
</ol>
<p>This is where my problem begins. On the project page, it now automatically creates a pull request card, but it already had the issue card. I somewhat understand why this is because they are 2 separate things.</p>
<p>However, for usability I don’t want to different card’s describing the same functionality.</p>
<p>Am I doing something wrong?</p>

**Comment by mpboom** [comment]
<p>No, you’re not doing anything wrong. That’s just how the system works.</p>
<p>You could, however, make some kind of ‘Done’ column and then configure the project board in a way that closed issues will automatically move there. Then, you could just close the issue and it will be moved to the ‘Done’ column while you can further work on the PR.</p>
<p> Another method would be to have 2 different projects, one for the issues and one for the PR (for example in case you want the issue to be open a bit longer to, for example, evaluate the new feature with the customer).</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/reijnn">@reijnn</a> Another thing to note: If you don’t select a Project to associate the PR with when creating the PR, a card won’t be automatically generated for it in that Project specifically (e.g. put into “Triage” for that Project). So, leaving the “Project” option blank when opening a new PR would mean no card is automatically put into the Triage for that Project, though a general card will be created and available for <em>any</em> Project for that PR.</p>
<p>If you wanted to still associate the PR to the issue, you could just using <a href="https://help.github.com/articles/autolinked-references-and-urls/" rel="nofollow noopener">autolinking</a> and input the issue URL to the PR, or vise versa. In this way, you establish a connection between the issue and PR, but the issue card remains the only card generated and marked for triage specifically for the Project board in question.</p>
